docs/application/dotnet/guides/nui/xaml/xaml-overview.md

```diff
@@ -0,0 +1,26 @@
+# Tizen.NUI.Xaml Overview
+
+XAML stands for **Extensible Application Markup Language**. It is a simple declarative language based on XML. XAML uses tags to define objects. Tags can be nested within other tags to define objects within objects.
+
+NUI adopts and integrates Xamarin.Forms XAML as NUI XAML. NUI XAML supports almost all the features of Xamarin.Forms XAML along with some additional features that are briefly explained in the subsequent sections. However, to get more information about XAML, see the [eXtensible Application Markup Language (XAML)](https://docs.microsoft.com/en-US/xamarin/xamarin-forms/xaml/) guide.
```

Comment:
@nawab-samsung, here the usage is eXtensible Application Markup Language (XAML)  but in line 3 it is Extensible Application Markup Language

Reply:
@annie-samsung  I agree with you, the reason was that even in Microsoft docs, it was written in both forms, so I left it. 

And the page that the link is referring to, the heading is in same context. 
https://docs.microsoft.com/en-us/xamarin/xamarin-forms/xaml/

@Seoyeon2Kim I request you to kindly change it to Extensible Application Markup Language (XAML)